.github/workflows/build.yml

```diff
@@ -3,150 +3,25 @@ name: Builds
 on:
   workflow_dispatch:
     inputs:
-      alpine:
-        description: 'Use alpine to build a fully static executable for linux'
+      version:
+        description: 'HLS version'
+        required: true
+        type: string
+      tag:
+        description: 'HLS tag'
         required: false
-        default: 'false'
+        type: string
+      project:
+        description: 'Gitlab project number'
+        type: number
+        required: false
+        default: 1178
```

Comment:
This well need to be changed to `1180`, which is the project id of https://gitlab.haskell.org/haskell/haskell-language-server

I set this to my fork for testing.